CMakeLists.txt

```diff
@@ -45,7 +56,7 @@ xcheck_add_c_compiler_flag(-Wno-format-truncation)
 xcheck_add_c_compiler_flag(-funsigned-char)
 
 # ClangCL is command line compatible with MSVC, so 'MSVC' is set.
-if(MSVC)
+if(${is_msvc})
```

Comment:
The comment should probably be updated. Also, why write `${is_msvc}` here but plain `is_msvc` elsewhere?